website/docs/connect-data/how-to-guides/how-to-use-the-camera-image-widget-to-upload-download-images.md

```diff
@@ -2,159 +2,59 @@
-![](/img/uploads31.PNG)
-
-
-In the content body, add the following:
-
-```
-{
-	"data": "{{Image1.image}}"
-}
-```
-
-Once you have added all the required parameters:
-
-* Now, set the Image Widget's **`onClick`** event to **execute a query**, and choose your query.
-
-Your image is stored in the S3 database once you run this query. Let's look at how to fetch an image from the S3 database.
-
-## Download the image
-
-#### Fetch Single File
-
-  <VideoEmbed host="youtube" videoId="dVZEd8p0Y2c" /> 
-
-* Click the + icon next to the **queries/js** and choose your **S3 datasource.**
-* Rename the query.
-* From the Commands drop-down, Select the method **Read file.**
-
-You can pass the below parameters to **Read a file.**
-
-* **Bucket Name:** Name of the bucket where the image is stored.
-* **File Path**: Path of the image you want to fetch. ex. images/name.
-* **File Data Type**: You should select base64 to display the image.
-
-Once you have added all the required parameters:
-
-* Drag and drop the [Image widget](/reference/widgets/image) onto the canvas.
-* In the Image property pane, add:
-
-```
-{{<your_query_name>.data.fileData}}
-```
-
-#### Fetch all files
-
-  <VideoEmbed host="youtube" videoId="UzV5LZ0kvDQ" /> 
-
-* Click the + icon next to the **queries/js** and choose your **S3 datasource.**
-* Rename the query.
-* From the Commands drop-down, Select the method **List files** in the bucket.
-* Add the **bucket name.**
-* Now, run the query.
-
-Now, open the query window and select the table option on the right-side property pane. It would automatically add a table widget to your canvas.
-
-:::info
-Bind the query’s response to the Table using JavaScript in the Table Data Property **`{{list_files.data}}`**.
-:::
-
-Now your table should list all the files present in your S3 bucket.
-
-You can use an image widget to display images listed in the table widget. You can follow this [guide](/connect-data/how-to-guides/how-to-upload-to-s3)to learn more.
-
-#### Download Files
-
-* Open the image property pane.
-* Click the **JS** button next to the **`onClick`** event and write the following JavaScript query:
-
-```
-{{download(atob(Fetch_image.data.fileData),'Testname','image/png')}}
-```
-
-Now, your image is downloaded when you click on the image widget.
-
-:::info
-You can check this [Guide](/connect-data/how-to-guides/how-to-upload-to-s3)to learn more about Upload/Download Files from S3.
-:::
-
-With Appsmith S3 integration, it is possible to create apps that seamlessly connect with the S3 database and provide additional flexibility for updating and analyzing data.
-
+# Upload Images Using Camera
+Appsmith lets you capture images and videos from your applications and share the data for further use.
+For more information, see [Camera](https://docs.appsmith.com/reference/widgets/camera).
+
+This page provides steps to upload images using the Camera widget to a datasource and uses the S3 datasource as an example.
```

Comment:
The guide will elaborate the steps for uploading images to s3 using the camera widget. Hence, **You will learn how to:** section is not needed.

Reply:
Done.